fabric-renderer-indigo/src/main/java/net/fabricmc/indigo/renderer/render/ItemRenderContext.java

```diff
@@ -202,19 +202,17 @@ private void renderQuad() {
         
         handleShading();
         
-        // A bit of a hack - copy packed normals on top of lightmaps.
-        // Violates normal encoding format but the editor content will be discarded
-        // and this avoids the step of copying to a separate array.
-        quad.copyNormals(quadData, EncodingFormat.VERTEX_START_OFFSET);
+        quad.populateMissingNormals();
+        quad.lightmap(15728880, 15728880, 15728880, 15728880);
```

Comment:
hex please, better static final constant w. hex value

Reply:
Was copying vanilla values but you're correct.  Will fix.